HMS.Client/src/app/features/dashboard/home/home.component.ts

```
@@ -1,10 +1,481 @@
 
+  completedAppsChart = new Chart({
+    chart: {
+      type: 'pie',
+      renderTo: 'appointments-donut',
+      backgroundColor: 'transparent',
+      events: {
+        render: function () {
+          const chart: any = this
+          const percentage = '77%';
+
+          if (chart.customLabel) {
```

Biome (1.9.4)

[error] This aliasing of this is unnecessary.

Arrow functions inherits `this` from their enclosing scope.
Safe fix: Use this instead of an alias.


(lint/complexity/noUselessThisAlias)